pyproject.toml

```diff
@@ -12,7 +12,7 @@ build-backend = 'mesonpy'
 requires = [
     "meson-python>=0.9.0",
     "Cython>=0.29.32",
-    "pybind11>=2.10.0",
+    "pybind11>=2.10.0,!=2.10.2",
```

Comment:
when backported, this would presumably become: `pybind11>=2.10.0,!=2.10.2,<2.11.0`

Reply:
I'd just make that `pybind11>=2.10.0,<2.11.0`

Reply:
Just to avoid that they don't fix it in `2.10.3` - that would be a problem

Reply:
I just did what I said originally, but I can always do an RC3 or whatever if we're genuinely worried they won't fix it.

Reply:
That shouldn't require a whole new RC. Let's see if it gets fixed or at least investigated in time, and if not then you can decide to tweak the constraint for the final release.